src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.onclick = () => {
+    resolve();
+  };
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(reject, 3000);
+});
+
+promise1.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
+);
+
+promise2.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
+);
+
+function myDisplayer() {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+
+  div.textContent = 'Promise was resolved!';
+  body.appendChild(div);
+}
+
+setTimeout(myDisplayer2, 3000);
```

Comment:
The `setTimeout` function is used here to call `myDisplayer2` after 3000 milliseconds. This might not be necessary if the intention is to handle promise rejection using the `promise2` rejection handler. Consider removing this line if it conflicts with the promise logic.